knx/dpt/formats.go

```diff
@@ -30,6 +30,40 @@ func unpackB1(data []byte, b *bool) error {
 	return nil
 }
 
+func packB4(b0 bool, b1 bool, b2 bool, b3 bool) byte {
```

Comment:
Why not accept a value of  `[4]bool` instead?

Reply:
Totally makes sense. Will fix it.